sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -927,8 +928,10 @@ SUBROUTINE MDLFLD
            (IGET(752)>0).OR.(IGET(754)>0).OR.      &
            (IGET(278)>0).OR.(IGET(264)>0).OR.      &
            (IGET(450)>0).OR.(IGET(480)>0).OR.      &
+           (IGET(479)>0).OR.(IGET(481)>0).OR.      &
            (IGET(774)>0).OR.(IGET(747)>0).OR.      &
            (IGET(464)>0).OR.(IGET(467)>0).OR.      &
+           (IGET(470)>0).OR.(IGET(476)>0).OR.      &
            (IGET(629)>0).OR.(IGET(630)>0).OR.      &
            (IGET(470)>0).OR.                       &
```

Comment:
@YaliMao-NOAA You may remove this line since "IGET(470)>0" is available in line 934.

Reply:
Thank you! Good catch.